tests/.pug-lintrc

```diff
@@ -0,0 +1,19 @@
+{
+"disallowAttributeInterpolation": true,
+"disallowClassAttributeWithStaticValue": true,
+"disallowDuplicateAttributes": true,
+"disallowHtmlText": true,
+"disallowIdAttributeWithStaticValue": true,
+"disallowLegacyMixinCall": true,
+"disallowMultipleLineBreaks": true,
+"disallowSpacesInsideAttributeBrackets": true,
+"requireClassLiteralsBeforeAttributes": true,
+"requireIdLiteralsBeforeAttributes": true,
+"requireLowerCaseAttributes": true,
+"requireLowerCaseTags": true,
+"requireSpaceAfterCodeOperator": true,
+"requireStrictEqualityOperators": true,
+"validateAttributeSeparator": { "separator": ", ", "multiLineSeparator": ",\n    " },
+"validateDivTags": true,
+"validateTemplateString": true
+}
```

Comment:
Could we move this to the project root or add the `pugLintConfig` key to the `package.json` as described in the [documentation](https://github.com/pugjs/pug-lint#configuration-file) so that editors will pick up the config?